components/gitpod-db/src/typeorm/entity/db-team-member.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { Entity, Column, PrimaryColumn, Index } from "typeorm";
+import { TypeORM } from "../typeorm";
+
+@Entity()
+// on DB but not Typeorm: @Index("ind_lastModified", ["_lastModified"])   // DBSync
+export class DBTeamMember {
```

Comment:
Can we call it Membership? Member seems to denote a person not the relation between team and user.

Reply:
Good point, thanks! Will adjust.